### PR TITLE
Settings profile screen

### PR DESCRIPTION
**Pull request by Viacheslav83**

### CODE REVIEW

File: INTITA/Modules/ProfileSettings/Cells/HeaderSettingsTableViewCell.swift [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+    
+    @IBAction func editButtonTapped(_ sender: UIButton) {
+        
+        delegate.editTaped(self)
+                
+    }
+    
+    static func nib() -> UINib {
+        return UINib(nibName: identifire, bundle: nil)
+    }
```
Comment (spanastasia) [inline INTITA/Modules/ProfileSettings/Cells/HeaderSettingsTableViewCell.swift:40]: `static func nib() -> UINib` and `static let identifire = "HeaderSettingsTableViewCell"`
were created twice in different classes. Maybe, it makes sense to take it out to protocol or extention.

Comment (oleksiikolomiietssnapp) [reply INTITA/Modules/ProfileSettings/Cells/HeaderSettingsTableViewCell.swift:40]: I'm agree on that

**Review by oleksiikolomiietssnapp — APPROVED** [review]

File: INTITA/Modules/Profile/ProfileViewController.swift [hunk 2]
```diff
@@ -297,7 +301,8 @@ extension ProfileViewController: ProfileViewModelDelegate {
 
 extension ProfileViewController: ProfileCoordinatorAlertPresenter {
     func showAlert() {
-        showAlert(header: "Oops!", message: "comming_soon".localized)
+//        coordinator?.settingScreen()
+//        showAlert(header: "Oops!", message: "comming_soon".localized)
```
Comment (oleksiikolomiietssnapp) [inline INTITA/Modules/Profile/ProfileViewController.swift:305]: Please take a look if it is needed to be commented

**Review by oleksiikolomiietssnapp — APPROVED** [review]

File: INTITA/Modules/Profile/ProfileCoordinator.swift [hunk 3]
```diff
@@ -38,6 +38,24 @@ class ProfileCoordinator: Coordinator {
...
+    
+    func displaySettingsProfileScreen() {
+        
+        let settingScreenCoordinator = SettingsProfileCoordinator(navigationController: navigationController)
+        
+        
+        settingScreenCoordinator.start()
+    }
+    
+    func removeCoordinator(_ coordinator: Coordinator) {
```
Comment (svitlana-korostelova) [inline INTITA/Modules/Profile/ProfileCoordinator.swift:50]: We already don't need this function, remove it please

Comment (oleksiikolomiietssnapp) [reply INTITA/Modules/Profile/ProfileCoordinator.swift]: fixed

File: INTITA/Modules/Profile/ProfileCoordinator.swift [hunk 4]
```diff
@@ -23,7 +23,7 @@ class ProfileCoordinator: Coordinator {
         //childCoordinators.append(MessagesCoordinator())
         //childCoordinators.append(OpportunitiesCoordinator())
         //childCoordinators.append(TasksCoordinator())
-        //childCoordinators.append(SettingsCoordinator())
+//        childCoordinators.append(SettingsProfileCoordinator(navigationController: navigationController))
```
Comment (svitlana-korostelova) [inline INTITA/Modules/Profile/ProfileCoordinator.swift:26]: All comments ready to be deleted

**Review by svitlana-korostelova — APPROVED** [review]

**Pull request merged**